### ISSUE TITLE
DADAgger: Disagreement-Augmented Dataset Aggregation. (arXiv:2301.01348v1 [cs.LG])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#24238]
DADAgger: Disagreement-Augmented Dataset Aggregation. (arXiv:2301.01348v1 [cs.LG])<br>
https://ift.tt/aRvrO6K<br>
<p>DAgger is an imitation algorithm that aggregates its original datasets by querying the expert on all samples encountered during training. In order to reduce the number of samples queried, we propose a modification to DAgger, known as DADAgger, which only queries the expert for state-action pairs that are out of distribution (OOD). OOD states are identified by measuring the variance of the action predictions of an ensemble of models on each state, which we simulate using dropout. Testing on the Car Racing and Half Cheetah environments achieves comparable performance to DAgger but with reduced expert queries, and better performance than a random sampling baseline. We also show that our algorithm may be used to build efficient, well-balanced training datasets by running with no initial data and only querying the expert to resolve uncertainty.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
